flatnav/util/SimdBaseTypes.h

```diff
@@ -134,31 +134,44 @@ struct simd256bit {
   __m128 get_low() const { return _mm256_extractf128_ps(_float, 0); }
   __m128 get_high() const { return _mm256_extractf128_ps(_float, 1); }
 
-  void loadu(const void *pointer) {
+  inline void loadu(const void *pointer) {
     _float = _mm256_loadu_ps((const float *)pointer);
   }
 
-  // This is supposed to be faster than using _mm256_store_ps to store
-  // a simd256bit type into a float array of size 8 and then performing
-  // scalar addition.
-  // TODO: Actually run some benchmarks to verify this.
+  /**
+   * Computes the sum of all numbers in a SIMD register using AVX instructions.
+   * This is actually a bit faster than doing the following
```

Comment:
Actually, I noticed that it is not always faster for reasons that I don't understand. For consistency, in the code that computes the distance I'm not actually using this reduce_add function. I'm doing the manual addition of eight floats.